src/vuestic-theme/vuestic-components/va-slider/VaSlider.vue

```diff
@@ -0,0 +1,582 @@
+        return a.some((v, i) => v !== b[i])
+      }
+      return a !== b
+    },
+    validateBorders () {
+      let isRightBorders = true
+
+      if (this.max < this.min) {
+        console.error('The maximum value can not be less than the minimum value.')
+        isRightBorders = false
+      }
+
+      if (this.min > this.max) {
+        console.error('The minimum value can not be greater than the maximum value.')
+        isRightBorders = false
+      }
+
+      return isRightBorders
+    },
+    validateValue (val) {
+      let isRightValue = true
+
+      const inRange = (v) => {
+        if (v < this.min) {
+          console.error(`The value of the slider is ${v}, the minimum value is ${this.min}, the value of this slider can not be less than the minimum value`)
+          isRightValue = false
+        } else if (v > this.max) {
+          console.error(`The value of the slider is ${v}, the maximum value is ${this.max}, the value of this slider can not be greater than the maximum value`)
+          isRightValue = false
+        }
+      }
+
+      if (this.isRange) {
+        val.map((v) => inRange(v))
+      } else {
+        inRange(val)
+      }
+
+      return isRightValue
+    },
+    validatePins () {
+      if ((this.max - this.min) % this.step !== 0) {
+        console.error('Step is illegal. Slider is nondivisible.')
+        return false
+      }
+
+      return true
+    },
+  },
+  mounted () {
+    this.$nextTick(() => {
+      if (this.validateValue(this.value) && this.validateBorders() && this.validatePins()) {
+        this.isComponentExists = true
+        this.getStaticData()
+        this.bindEvents()
+      } else {
+        this.isComponentExists = false
+      }
+    })
+  },
+  beforeDestroy () {
+    this.isComponentExists = false
+    this.unbindEvents()
+  }
+}
+</script>
+
+<style lang='scss'>
+  $slider-colors: (
+    success: (#40e583, #d6ffd3),
+    danger: (#e34b4a, #ffebeb),
+    info: (#2c82e0, #caeeff),
+    warning: (#ffc200, #fff3d1)
+  );
+
+  @each $name, $colors in $slider-colors {
+    $active-track: nth($colors, 1);
+    $track: nth($colors, 2);
+
+    .va-slider--#{$name} {
+
+      .va-slider__label, .va-slider__invert-label, .va-slider__container__handler-value {
+        color: $active-track;
+      }
+
+      .va-slider__container__track--active, .va-slider__container__mark--active {
+        background: $active-track !important;
+      }
+
+      .va-slider__container__track, .va-slider__container__mark {
+        background: $track;
+      }
+
+      .va-slider__container__handler {
+        border: 0.375rem solid $active-track;
+      }
+    }
+  }
+
+  .va-slider {
+
+    &--disabled {
+      @include va-disabled;
+    }
+
+    &__label {
+      margin-right: 1rem;
+    }
+
+    &__invert-label {
+      margin-left: 1rem;
+    }
+
+    &__container {
+      position: relative;
+      width: 100%;
+      height: 0.5rem;
+
+      &__track {
+        position: absolute;
```

Comment:
if you use block with (display flex) as a parent there is no needed to use position absolute for a child